### PR TITLE
chore(deps): implement github workflows; rename "start" script to "watch"

### PR DESCRIPTION
**Pull request by dkoo**
* Renames `start` script to `watch` as [suggested here](https://github.com/Automattic/newspack-plugin/pull/1368#discussion_r769813823)
* Adds the `.github` workflows (labeling and Dependabot/auto-merge) and PR/issue templates we're using in other repos

I'm not sure how to test the script rename, but hopefully this illustrates what I mean.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+### All Submissions:
+
+* [ ] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
+* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
```
Comment (adekbadek) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
```

Nit: WP Core standards don't apply here.

**Review by adekbadek — APPROVED** [review]
👌

**Comment by dkoo** [comment]
Thanks, @adekbadek! Do we need to `npm publish` too?

**Comment by adekbadek** [comment]
> Do we need to npm publish too?

The 3263c1bb8cb60aa9b5d6993b73d835e0139b09e2 commit had a non-triggering commit message, and it should've also stated a breaking change. I've pushed an empty, breaking-release-triggering commit on `master` (114244358dc95da06aa9fd97eff0fb12f6cbc11d) and we're now at `v2.0.0`.

**Comment by dkoo** [comment]
Aha, thanks for the clarification!

**Pull request merged**